Resources/Prototypes/secret_weights.yml

```
@@ -4,7 +4,5 @@
     Nukeops: 0.20
     Traitor: 0.60
     Zombie: 0.04
```

```suggestion
    Zombie: 0.05
```
At least divide up the chance nicely. A 4 and 11% chance is annoying me way too much 😄

---

Increasing zombie chance??? Evil man, you are...
Pushed requested changes.

---

The 1 additional percent of zombies was already in zombieteors, so this keeps is the same actually.